CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Improved
 - An error will be thrown when attempting to `release` from a repository with un-pushed local changes. #TINY-7407
+- The `release` command will now fail if the package still contains pre-release dependencies. #TINY-7502
```

Comment:
Should there be an `### Added` for the new option as well?

Reply:
```suggestion
- The `release` command will now fail if the package still contains pre-release dependencies. This can be disabled using the `--allow-pre-releases` option. #TINY-7502
```